instrumentation/racecar/lib/opentelemetry/instrumentation/racecar/instrumentation.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'opentelemetry-instrumentation-rdkafka'
+
+module OpenTelemetry
+  module Instrumentation
+    module Racecar
+      # The Instrumentation class contains logic to detect and install the Racecar instrumentation
+      class Instrumentation < OpenTelemetry::Instrumentation::Base
+        MINIMUM_VERSION = Gem::Version.new('2.0')
+
+        compatible do
+          Gem::Version.new(::Racecar::VERSION) >= MINIMUM_VERSION
```

Comment:
Nit but would you mind abstracting this gem version check into a private method `gem_version`, which `compatible` then calls ? This follows the style we've tried to implement in this pr https://github.com/open-telemetry/opentelemetry-ruby-contrib/commit/3b03ff7ea66b69c85ba205a369b85c2c33b712fe

see example https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/26a9b3b7cc6e37f71b29087ee06639aad6cd459a/instrumentation/pg/lib/opentelemetry/instrumentation/pg/instrumentation.rb#L50-L52

Reply:
done